woocommerce/woocommerce-admin
# Migrate onboarding data store to wp data

Moves the onboarding data store towards wp data and away from the wc-api.

Note that this adds a check to all wp-data store registrations to prevent duplicate registration of stores until the webpack chunk issue is resolved.  See p1590435699034400-slack-woo-start-manage

### Detailed test instructions:

1. Disable WCS and/or Jetpack.
1. Walk through the dashboard, profiler, and task list.
1. Make sure no errors occur.
1. Make sure profile data is persisted across page loads (try refreshing mid profiler to make sure profile items are hydrated on load).
